app/Repositories/Eloquents/NotificationEloquentRepository.php

```diff
@@ -84,11 +89,27 @@ public function find($id)
     public function update($data = [])
     {
         try {
-            $record = $this->model()->findOrFail($data['id']);
-        
-            return $record->update($data);
+            if (array_has($data, 'id')) {
+                $record = $this->model()->findOrFail($data['id']);
+
+                return $record->update($data);
+            } else {
+                return null;
+            }
+            
         } catch (Exception $e) {
             return null;
         }
     }
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/NotificationEloquentRepository.php</br>Blank line found at end of control structure